monix-eval/shared/src/test/scala/monix/eval/TaskGatherUnorderedSuite.scala

```diff
@@ -134,12 +134,24 @@ object TaskGatherUnorderedSuite extends BaseTestSuite {
 
     val ex = DummyException("dummy1")
     var errorsThrow = 0
-    val task1 = Task.raiseError[Int](ex).executeAsync.doOnFinish { x =>
-      if (x.isDefined) errorsThrow += 1; Task.unit
-    }
-    val task2 = Task.raiseError[Int](ex).executeAsync.doOnFinish { x =>
-      if (x.isDefined) errorsThrow += 1; Task.unit
-    }
+
+    val task1 = Task
+      .raiseError[Int](ex)
```

Comment:
Is it scalafmt style? That separation of companion object and a factory method on different lines kinda bothers me

Reply:
Unfortunately yes. Not sure what to do about it.

Reply:
Huh, that's weird. I think you can do:
```
optIn {
  breaksInsideChains = false
  breakChainOnFirstMethodDot = false //pretty sure it's this one causing this.
}
```